server.py

```diff
@@ -159,6 +159,13 @@ def format_raw_course(raw_course):
 def schedule_sort_key(slot):
     return slot["days"], slot["startTime"], slot["endTime"]
 
+def unique_schedule(schedule):
+    new_schedule = []
+    for time_slot in schedule :
+        if time_slot not in new_schedule :
+            new_schedule += [time_slot]
+    return new_schedule
+
```

Comment:
Can we just make this a generic utility function? There's no reason it should be limited to operation on lists of time slots; it should work to deduplicate any list of non-hashable objects.

Reply:
That's true. I will fix that.